app/build.gradle

```
@@ -11,7 +11,12 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
     buildTypes {
+        debug {
+            debuggable true
+            buildConfigField 'String', "BASE_URL", githubBaseUrl
```

Esto podría variar. Se pueden guardar las URLs en archivos de recursos Strings.xml para ser ofuscados y colocarlos dependiendo el _flavor_ y _build variant_

---

Genial, lo revisare :+1: 

```suggestion
            buildConfigField 'String', "BASE_URL", githubBaseUrl
```